messenger/handler_results.rst

```diff
@@ -0,0 +1,124 @@
+.. index::
+    single: Messenger; Getting results / Working with command & query buses
+
+Getting results
+---------------
+
+When a message is handled, the :class:`Symfony\\Component\\Messenger\\Middleware\\HandleMessageMiddleware`
+adds a :class:`Symfony\\Component\\Messenger\\Stamp\\HandledStamp` for each responding handler,
+which can be used to get their returned value:
+
+.. configuration-block::
+
+    .. code-block:: php
+
+        use Symfony\Component\Messenger\MessageBusInterface;
+        use Symfony\Component\Messenger\Stamp\HandledStamp;
+
+        $envelope = $messageBus->dispatch(SomeMessage());
+
+        // Get the last handled stamp and its returned value:
+        $handledStamp = $envelope->last(HandledStamp::class);
+        $handledStamp->getResult();
+
+This assumes there is only one configured handler or that only the last handler
```

Comment:
I think "Using the `last` method" or something similar instead of "This" would be more clear.